android/app/src/main/java/app/organicmaps/bookmarks/data/Metadata.java

```diff
@@ -67,7 +67,9 @@ public enum MetadataType
     FMD_WEBSITE_MENU(46),
     FMD_SELF_SERVICE(47),
     FMD_OUTDOOR_SEATING(48),
-    FMD_NETWORK(49);
+    FMD_NETWORK(49),
+    FMD_CONTACT_FEDIVERSE(50),
+    FMD_CONTACT_BLUESKY(51);
```

Comment:
@pastk it should be added in general too.

Reply:
@rtsisyk do you mean in the Qt UI/editor? It is:

<img width="400px" src="https://github.com/user-attachments/assets/4534345a-8c32-4c11-8a32-2d7952bdb23f"/>

Reply:
@rtsisyk do you mean making an OM Bluesky account?